jobs/delorean/jenkinsfiles/discovery/ga/github/Jenkinsfile

```diff
@@ -169,7 +178,8 @@ def processInstallationBranch(body) {
                         gitCommitWhenChanges("Add ${product} to update products list") { msgs ->
                             def upgradeVariables = readYaml file: installationUpgradeFile
                             def upgradeProductRoles = upgradeVariables['upgrade_product_roles']
-                            upgradeProductRoles << product
+                            upgradeProductRoles.remove(product)
```

Comment:
This can be removed at some point, but for now we can leave it to deal with others that might need updated, and have it automatically fix it.